### ISSUE TITLE
[BUG] Playlist Repeat Does Not Save

### ISSUE THREAD

**Issue by mermelmadness** [Aircoookie/WLED#3058]
### What happened?

I built a playlist and set it to repeat once since I only want it to run once when activated.  Every time I save it runs once, but when I want to change something the repeat changes to anything between 2 and 9.  Rebooting does not fix the issue.

### To Reproduce Bug

Make a playlist and set it to run once.  The save and let it run.  After, go back to edit it and see what the repeat is set to.

### Expected Behavior

I would expect the repeat setting to remain whatever it is set to initially.

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.14.0-b2 2301240

### Which microcontroller/board are you seeing the problem on?

ESP32

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mermelmadness** [comment]
This is still an issue.
![image](https://user-images.githubusercontent.com/38080873/216409071-326db6ec-dc2c-4917-99f3-ca3fd50d0d40.png)
![image](https://user-images.githubusercontent.com/38080873/216409155-9d82c87c-5e2e-4e65-824e-c852e22b0265.png)

**Comment by blazoncek** [comment]
I can see wrong repeat count on 1st save but subsequent saves are ok.
Next time please describe a problem in more detail.

**Comment by mermelmadness** [comment]
I am not seeing that behavior.  If I uncheck Repeat Indefinitely and set repeat to 1, then save, then immediately click the playlist dropdown, Repeat Indefinitely is checked again.  Restarting does not fix the issue.

**Comment by dosipod** [comment]
Old behavior is playlists repeat will default to 2 even if you try to make an effect run only once
![image](https://user-images.githubusercontent.com/87589371/216865476-05498862-94ec-4be0-befc-82d13c9435d9.png)

**Comment by blazoncek** [comment]
I am not sure I understand what you are saying, so please open a new issue describe it again in every possible detail with examples what you think is wrong and how you think it should work if the problem still persists.
Please use most recent source code as there were some updates. Do not use precompiled binaries available on https://install.wled.me until a new beta is released.

I have tested the recent code and I am able to save repetition count of 1 or more including indefinite repetitions.

**Comment by alvindotdev** [comment]
I'm seeing the same issue as last published. 

1. I created a playlist containing several presets
2. I do not want it to repeat indefinitely - i do not want it to repeat at all. so i uncheck that box and set repeat to 1 time
3. I save
4. Immediately the playlist runs on my led strip without repeating
5. If i now expand the playlist again to see the settings, repeat is set to (2)
6. If I view the api call, i can see repeat:2 in the json
7. If i play the playlist anytime going forward, it will repeat twice

I can't seem to permanently make the playlist stay at repeat 1. Even if i update the API call manually to set repeat:1, then save, it will still do the same steps 4-7 above.

This occurs in both the iphone app and the web browser mode

Setting repeat to (1)
![Screen Shot 2023-02-23 at 4 02 28 PM](https://user-images.githubusercontent.com/60757584/221059426-07a99dde-8ed5-4de1-b044-8c3ca939a10d.png)

Save and it is not expanded any longer -- it will run the playlist automatically and not repeat which is GOOD
![Screen Shot 2023-02-23 at 4 02 44 PM](https://user-images.githubusercontent.com/60757584/221059454-4877fb1d-b06e-43c0-8015-ab95b2188f3e.png)

Any subsequent play to the playlist will repeat (2) times, and when i expand to edit, (2) will be written in the playlist repeat setting
![Screen Shot 2023-02-23 at 4 02 49 PM](https://user-images.githubusercontent.com/60757584/221059531-95fd2f00-1a13-474a-90a0-a6c99d669539.png)

![Screen Shot 2023-02-23 at 4 02 57 PM](https://user-images.githubusercontent.com/60757584/221059551-1ffee92f-c853-45f4-a63f-6d31e0a54f1b.png)


Even if i edit the api command directly to repeat:1...
![Screen Shot 2023-02-23 at 4 07 27 PM](https://user-images.githubusercontent.com/60757584/221059597-9507e8a7-983c-4131-90de-cf3256fe3797.png)


It will do the same things described above --run without repeat successfully 1 time , and subsequent runs will repeat:2.


![Screen Shot 2023-02-23 at 4 08 25 PM](https://user-images.githubusercontent.com/60757584/221059715-6c4120b1-9d36-4019-84e8-a04ba453a931.png)

I just flashed wled 1 day ago on the latest version on a new ESP32

**Comment by dosipod** [comment]
This post is closed , might be best copy your input into a new post

**Comment by degracode** [comment]
Was a new issue opened for this @alvindotdev? I'm seeing this problem exactly as you describe; I change to 1 repeat, and it reverts back to 2.

0.14.0-b1 (build 2212222)

**Comment by blazoncek** [comment]
> 0.14.0-b1 (build 2212222)

Use 0.14.1-b2

**Comment by degracode** [comment]
> > 0.14.0-b1 (build 2212222)
> 
> Use 0.14.1-b2 or newer.

Are you confirming that a problem was identified and fixed, or just that I should speculatively give the new version a try?
